WordPress/src/main/java/org/wordpress/android/ui/posts/PublishSettingsViewModel.kt

```diff
@@ -53,7 +53,9 @@ constructor(
     val onPostStatusChanged: LiveData<PostStatus> = _onPostStatusChanged
     private val _onUiModel = MutableLiveData<PublishUiModel>()
     val onUiModel: LiveData<PublishUiModel> = _onUiModel
-    private val _onToast = MutableLiveData<Event<String>>()
+
+    @Suppress("VariableNaming")
+    protected val _onToast = MutableLiveData<Event<String>>()
```

Comment:
If you would like to avoid @Suppress then you could name it something like **onToastEvent** perhaps

Reply:
Detekt gives an error when a **protected** variable starts with **_**, but it doesn't give an error for a **private** variable. I could add a new variable like `protected val onToastEvent` but I couldn't decide which one is better, suppressing or adding a new variable.